compiler/one-cmds/utils.py

```diff
@@ -81,18 +81,19 @@ class _CONSTANT:
 _CONSTANT = _CONSTANT()
 
 
-def _add_default_arg(parser):
+def _add_default_arg(parser, support_config=True):
     # version
     parser.add_argument(
         '-v',
         '--version',
         action='store_true',
         help='show program\'s version number and exit')
 
-    # configuration file
-    parser.add_argument('-C', '--config', type=str, help='run with configuation file')
-    # section name that you want to run in configuration file
-    parser.add_argument('-S', '--section', type=str, help=argparse.SUPPRESS)
+    if support_config is True:
+        # configuration file
+        parser.add_argument('-C', '--config', type=str, help='run with configuation file')
+        # section name that you want to run in configuration file
+        parser.add_argument('-S', '--section', type=str, help=argparse.SUPPRESS)
```

Comment:
I don't know about details of not supporting config file. But isn't it needed?
```yaml
[one-profile]
backend=dummy
command=sample.tvn
```
Hmmm.. on second thought, since profile is not the process of compiling, it seems not be needed.

Reply:
@mhs4670go For limit of time (due to Aug 5), we've decided to existing features only. Configuration for `one-profile` will be the next step after completing skeleton of `onecc` within Aug 5.